src/internal.rs

```diff
@@ -805,7 +805,7 @@ pub(crate) mod tests {
     use vec1::vec1;
 
     /// String contains matcher to assert that the provided substring exists in the provided value
-    pub fn contains<'a>(expected: &'a str) -> Box<dyn Matcher<String> + 'a> {
+    pub fn contains(expected: &str) -> Box<dyn Matcher<String> + '_> {
```

Comment:
Also weird - does it now infer the lifetime somehow?

Reply:
Same